src/Mvc/Mvc.Core/test/Authorization/AuthorizeFilterTest.cs

```diff
@@ -527,6 +512,89 @@ public void CreateInstance_ReturnsNewFilterIfPolicyAndPolicyProviderAreNotSet(Au
             Assert.Same(policyProvider, actual.PolicyProvider);
         }
 
+        [Fact]
+        public async Task GetEffectivePolicyAsync_ReturnsCurrentPolicy_WhenNoEndpointMetadataIsAvailable()
```

Comment:
A couple of unit tests to go through the motion of verifying the combining code works